### ISSUE TITLE
13200766: UIATableGroup accessibility identifier makes items inside inaccessible.

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#5293]
#### Description

Summary:
UIATableGroup accessibility identifiers are (randomly) inconsistant if the table group contains more than one label.

Steps to Reproduce:
Create a UITableView with a custom UIView returned as a section view.  In this section view, place a segmented control with textual options.  Also, place a UILabel in this section view that is meant to be the section heading.

Expected Results:
The section heading labels should be returned as the name of the UIATableGroup element on the UI Automation side.

Actual Results:
The name of one of the buttons in the segmented control are randomly returned as the name of the UIATableGroup.

Notes:
A work around for this would simply be to put an accessibility identifier on the UIView that we return for the table group section.  However, in attempting to do so, the interior contents of that custom section view are inaccessible from automation.  Therefor, for example, we cannot deterministically access a table group element and click on one of it's child elements, because those child elements no longer show up.
## 

Product Version: 4.6
Created: 2013-02-12T20:27:26.464554
Originated: 2013-02-12T00:00:00
Open Radar Link: http://www.openradar.me/13200766
